inc/Bed.h

```diff
@@ -0,0 +1,19 @@
+#include "Client.h"
+#include "Module.h"
+
+class Bed : public Module
+{
+private:
+  bool Led;
+  bool Switch;
+  int DB; // druk sensor dit is analoog
```

Comment:
Dutch comments should be switched to English.
Variable names shouldn't start with an capitalized letter. Project uses lowerCamelCase.